### DISCUSSION TITLE
[Example] 캐릭터 컨트롤러 커스터마이즈 가이드 및 샘플

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#826] [2. Official Samples]
안녕하세요, 제페토 에반젤리스트입니다.
캐릭터 컨트롤러 커스터마이즈 가이드 및 샘플이 업데이트 되었습니다 😄

### [Character Controller Customize]
https://studio.zepeto.me/kr/guides/character-controller-customize

- Character Controller Customize
- Double Jump Setting
- FPS/TPS View Example
- Quarter View Example
- Side View Example
 
### [Character Controller Customize Samples]
기존 멀티플레이 샘플에 추가되었습니다. 
샘플을 바탕으로 멋진 월드를 만들어주시기를 기대하고 응원합니다 💪

**TPS View**
https://github.com/naverz/zepeto-multiplay-example/tree/main/Assets/Scenes/TPSView
![8-1](https://user-images.githubusercontent.com/100118650/171150131-fe7a9a51-34b0-4ca9-9814-bf43abb63602.gif)

**Quearter View**
https://github.com/naverz/zepeto-multiplay-example/tree/main/Assets/Scenes/QuarterView
![11-1](https://user-images.githubusercontent.com/100118650/171150179-a7644b5f-7420-43c5-b517-4a421b19c1ea.gif)

**Side View**
https://github.com/naverz/zepeto-multiplay-example/tree/main/Assets/Scenes/SideView
![16-1](https://user-images.githubusercontent.com/100118650/171150200-55d67072-6891-47e3-8a3c-506581a2110d.gif)

**Comment by minseon-park** [comment]
멋진 샘플 감사합니다. 맵 안에서 건물로 들어갈때의 시야변경은 보편적으로 TPSview 를 사용하는것인가요?  Ex) 지하철맵에서 CU 편의점을 들어갔을때 카메라 시야 변경

**Reply by Zepeto-TechEvangelist** [reply]
네 맞습니다 😄
